portfolio.html

```diff
@@ -139,6 +139,585 @@ <h2 class="description">I'm a software developer</h2>
           </div>
 
     </section>
+    <div class="down-arrow">
+      <svg
+        width="42"
+        height="42"
+        viewBox="0 0 42 42"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <rect x="1" y="1" width="40" height="40" rx="20" fill="white" />
+        <path
+          fill-rule="evenodd"
+          clip-rule="evenodd"
+          d="M14.2929 17.2929C14.6834 16.9024 15.3166 16.9024 15.7071 17.2929L21 22.5858L26.2929 17.2929C26.6834 16.9024 27.3166 16.9024 27.7071 17.2929C28.0976 17.6834 28.0976 18.3166 27.7071 18.7071L21.7071 24.7071C21.3166 25.0976 20.6834 25.0976 20.2929 24.7071L14.2929 18.7071C13.9024 18.3166 13.9024 17.6834 14.2929 17.2929Z"
+          fill="#172B4D"
+        />
+        <rect x="1" y="1" width="40" height="40" rx="20" stroke="#DFE1E6" />
+      </svg>
+    </div>
+  </section>
+  <section class="work-section">
```

Comment:
I suggest you fix the margins of your project to match figma design. Its part of the project requirements. Currently your cards are smaller in size and leave a lot of white space. For you to pass this milestone, please fix this

- [ ] This is a screenshot for your project currently
<img width="460" alt="Screenshot 2022-10-01 at 18 14 38" src="https://user-images.githubusercontent.com/48546992/193416245-f23a8753-e797-420c-b229-b5445fa6ee33.png">

- [ ] This is a screenshot of the figma design
<img width="371" alt="Screenshot 2022-10-01 at 18 14 48" src="https://user-images.githubusercontent.com/48546992/193416266-a6257cc2-6aac-452a-9e53-afd1777d4358.png">